exercises/darts/canonical-data.json

```diff
@@ -0,0 +1,56 @@
+{
+  "exercise": "darts",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Return the correct amount earned by a dart landing in a given point in the target problem.",
+      "cases": [
+        {
+         "property": "score",
+         "description": "A dart lands outside the target",
+         "input": {
+           "x": 15.3,
+           "y": 13.2
+         },
+         "expected": 0
+        },
+        {
+         "property": "score",
+         "description": "A dart lands just in the border of the target",
+         "input": {
+           "x": 10,
+           "y": 0
+         },
+         "expected": 1
+        },
+        {
+         "property": "score",
+         "description": "A dart lands in the middle circle",
+         "input": {
+           "x": 3,
+           "y": 3.7
+         },
+         "expected": 5
+        },
+        {
+         "property": "score",
+         "description": "A dart lands right in the border between outside and middle circles",
+         "input": {
+           "x": 0,
+           "y": 5
+         },
+         "expected": 5
+        },
+        {
+         "property": "score",
+         "description": "A dart arrives in the inner circle",
```

Comment:
In the other test cases "lands" is used instead of "arrives". Maybe make this consistent?